### ISSUE TITLE
Jest indicated that certain test suites are failing to run

### ISSUE THREAD

**Issue by chrismgonzalez** [OperationCode/front-end#155]
# Bug Report
## What is the current behavior?
Jest is throwing errors that test suites are failing to run in certain files.
## What is the expected behavior?
Jest should not throw errors in test.js files.

## What steps did you take to get this behavior?
Ran the test runner `yarn test`
### Operating System
Windows 10 Pro Version 1803  OS build 17134.285
### Screenshots
![jest error](https://user-images.githubusercontent.com/10368310/45921000-89aa7900-be72-11e8-94ea-825083539515.png)

**Comment by nvwebd** [comment]
Hi guys / gals.

Just took a look at this and I'm finding it hard to follow the chain of though:

- what branch are you running this on?
- if you check the error stack it says that an err occurs in jest.setup.js and this state is not on master

Branch out from your state and push to a feature/??? branch so we can have a closer look.

Best.

**Comment by kylemh** [comment]
I believe Chris encounters this on every branch. What do you mean by `state is not on master`?

**Comment by nvwebd** [comment]
Hi. What I mean that the master branch doesn't have the same source code state as Chris has.

But after inspection I've noticed that the code is the same :)

The error line threw me off a bit.

I have Bootcamp on my mac and I'll switch to windows and check it out.

**Comment by kylemh** [comment]
Oh yeah! Column number 17, not line number 17 - right? Threw me off for a second.

**Comment by chrismgonzalez** [comment]
I'm going to run the tests on my mac and report back.

**Comment by chrismgonzalez** [comment]
Tests on my mac and PC are identical.  I think the root cause of the problem is with the Jest extension I have installed.  

I'm OK with closing this issue for now.

**Comment by kylemh** [comment]
Can you try again when using the version of node indicated in `.nvmrc`?

**Comment by kylemh** [comment]
Check out https://github.com/coreybutler/nvm-windows

**Comment by chrismgonzalez** [comment]
My node version is the same as the one in `.nvmrc`.  The Jest extension works fine in my MacOS machine.  Still pooping out on Windows.  However, when I run `yarn test` or `yarn jest --coverage` I get the same readout in the console.

**Comment by kylemh** [comment]
Closing this. Hopefully it's isolated to the specific extension and some kind of incorrect config.
